Python2to3 changes part1

RichardWaiteSTFC opened the pull request:
Rebased this branch python2to3_changes_part1 (formerly called case_changed) with master and resolved conflicts due to contributors altering master since python 2&3 compatibility changes were made.

Commit:
Rename like names with different cases for windows compat

RichardWaiteSTFC committed:
Added a file after check that date function worked post 2to3.py

2to3.py adds "o" to date commands e.g. "0o3" for 03 which works in pyhton 2 and 3

RichardWaiteSTFC committed:
Removed double import print_function and double brakets after prints

Only print statements that were the only thing on that line were ammended (done by automated script)

RichardWaiteSTFC committed:
Manual correction of print brackets for unusual cases.

RichardWaiteSTFC committed:
Added line breaks between import and print to some test scripts

RichardWaiteSTFC committed:
Corrected more mis-matched brackets on print functions in one file

RichardWaiteSTFC committed:
Even more mis-matched print brackets corrected

martyngigg commented on direct_inelastic/MERLIN/calibration/create_MERInst_Files.py:
```diff
@@ -248,8 +249,8 @@ def create_one2onemap(one2oneold,one2onenew):
      # list of the known cycles with approximate  end dates:          
-    cycle_end_list = {date(2019,03,29):'184',date(2019,07,19):'191',date(2019,10,25):'192',\
-    date(2019,12,20):'193',date(2999,01,01):'future'}
+    cycle_end_list = {date(2019,0o3,29):'184',date(2019,0o7,19):'191',date(2019,10,25):'192',\
```
It looks like there are additional changes to the `date()` call - was this intended?

martyngigg commented on user/NickDraper/test script 2 copy.py:
```diff
@@ -1 +1,2 @@
-print "hi there"
\ No newline at end of file
+from __future__ import print_function)
```
I don't think the bracket at the end is meant to be there? Similarly for the files below?

martyngigg reviewed (changes requested):
A couple of things that I just want to clarify before we merge

RichardWaiteSTFC committed:
Python 2to3.py changes for compatibility

This commit contains the majority of the 2to3.py chnages but should have preserved the state of James Lord's scripts which he had already latered and tested

RichardWaiteSTFC committed:
Removed 0o in date of MER script and fixed typo in Nick Draper script

Changes requested in PR by Martyn
